test/style/expression/expression.test.cpp

```diff
@@ -29,18 +29,21 @@ TEST(Expression, IsExpression) {
 
     for(auto& entry : allExpressions.GetObject()) {
         const std::string name { entry.name.GetString(), entry.name.GetStringLength() };
-        if (name == "line-progress" ||
-            name == "feature-state" ||
-            name == "interpolate-hcl" ||
-            name == "interpolate-lab" ||
-            name == "format") {
-            // Not yet implemented
-            continue;
-        }
         JSDocument document;
         document.Parse<0>(R"([")" + name + R"("])");
-
         const JSValue* expression = &document;
+
+        // TODO:   "feature-state": https://github.com/mapbox/mapbox-gl-native/issues/12613
+        // TODO: "interpolate-hcl": https://github.com/mapbox/mapbox-gl-native/issues/8720
+        // TODO: "interpolate-lab": https://github.com/mapbox/mapbox-gl-native/issues/8720
+        // TODO:          "format": https://github.com/mapbox/mapbox-gl-native/issues/12612
+        if (name == "feature-state" || name == "interpolate-hcl" || name == "interpolate-lab" || name == "format") {
+            if (expression::isExpression(conversion::Convertible(expression))) {
+                ASSERT_TRUE(false) << "Expression name" << name << "is implemented - please update Expression.IsExpression test.";
+            }
+            continue;
+        }
+
```

Comment:
This doesn't seem related to the PR — perhaps committed accidentally?

Reply:
It is - line-progress was implemented already and this new logic helps cleaning up TODOs like this when stuff gets implemented.